android/src/main/java/com/oblador/keychain/KeychainModule.java

```diff
@@ -130,23 +131,24 @@ public class KeychainModule extends ReactContextBaseJavaModule {
   /** Name-to-instance lookup  map. */
   private final Map<String, CipherStorage> cipherStorageMap = new HashMap<>();
   /** Shared preferences storage. */
-  private final PrefsStorage prefsStorage;
+  private final KeyValueStorage blockStoreStorage;
   //endregion
 
   //region Initialization
 
   /** Default constructor. */
   public KeychainModule(@NonNull final ReactApplicationContext reactContext) {
     super(reactContext);
-    prefsStorage = new PrefsStorage(reactContext);
+    blockStoreStorage = new BlockStoreStorage(reactContext);
 
-    addCipherStorageToMap(new CipherStorageFacebookConceal(reactContext));
+    addCipherStorageToMap(new CipherStorageBase64());
+//    addCipherStorageToMap(new CipherStorageFacebookConceal(reactContext));
     addCipherStorageToMap(new CipherStorageKeystoreAesCbc());
 
     // we have a references to newer api that will fail load of app classes in old androids OS
-    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
-      addCipherStorageToMap(new CipherStorageKeystoreRsaEcb());
-    }
+//    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+//      addCipherStorageToMap(new CipherStorageKeystoreRsaEcb());
+//    }
```

Comment:
I commented
` //    addCipherStorageToMap(new CipherStorageFacebookConceal(reactContext));`
and
```
//    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
//      addCipherStorageToMap(new CipherStorageKeystoreRsaEcb());
//    }
```
as it was commented originally in our patch for the library.